website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -763,3 +763,44 @@ Views with this configuration will be able to select from objects in `project_1.
 #### Limitations
 
 The `grant_access_to` config is not thread-safe when multiple views need to be authorized for the same dataset. The initial `dbt run` operation after a new `grant_access_to` config is added should therefore be executed in a single thread. Subsequent runs using the same configuration will not attempt to re-apply existing access grants, and can make use of multiple threads.
+
+## Materialized view
+
+The BigQuery adapter supports [materialized views](https://cloud.google.com/bigquery/docs/materialized-views-intro) and refreshes them for every subsequent `dbt run` you execute. For more information, see [Refresh Materialized Views](https://cloud.google.com/bigquery/docs/materialized-views-manage#refresh) in the Google docs.
+
+Materialized views support the optional configuration `on_configuration_change` with the following values: 
+- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The following changes can be applied without the need to rebuild the materialized view:
+  - enable_refresh 
+  - refresh_interval_minutes 
+  - max_staleness
+- `skip` &mdash; allows runs to continue while also providing a warning that the model was skipped
+- `fail` &mdash; forces runs to fail if a change is detected in a materialized view 
+
+You can create a materialized view by editing _one_ of these files:
```

Comment:
@matthewshaver can we add line 779 - 806 into a partial (was snippet) and reuse this content for all three pages? This will help us maintain docs content and more update this section in the future. I think these lines are exactly the same, but I didn't do a diff, so 🤷🏻‍♀️